generator/src/main/java/com/linkedin/pegasus/generator/JavaDataTemplateGenerator.java

```diff
@@ -1336,6 +1336,11 @@ private JFieldVar generateSchemaField(JDefinedClass templateClass, DataSchema sc
     // Generate the schema field initialization
     schemaField.init(JExpr.cast(getCodeModel()._ref(schema.getClass()), parseSchemaInvocation));
 
+    // Using "dataSchema" as method name since "schema" conflicts with RecordTemplate::schema and "getSchema" conflicts
+    // with TyperefInfo::getSchema
+    final JMethod staticFieldsAccessor = templateClass.method(JMod.PUBLIC | JMod.STATIC, schema.getClass(), "dataSchema");
```

Comment:
make "dataSchema" as a constant?
But to be honest I would not say I like this new name since it introduced another name!  Don't you think so?